src/vs/workbench/browser/parts/editor/media/editortitlecontrol.css

```diff
@@ -45,3 +45,8 @@
 .monaco-workbench .part.editor > .content .editor-group-container > .title .breadcrumbs-below-tabs .breadcrumbs-control .monaco-breadcrumb-item:last-child .codicon:last-child {
 	display: none; /* hides chevrons when last item */
 }
+
+.monaco-workbench .part.editor > .content .title .monaco-progress-container {
+	top: unset;
```

Comment:
What is this change about? Can it have a comment to explain?

Reply:
This is reset styles about absolute position which is defined in src/vs/workbench/browser/media/part.css.
I have add a comment too. See the latest commit.